tools/server/tests/utils.py

```diff
@@ -379,6 +382,21 @@ def jina_reranker_tiny() -> ServerProcess:
         server.server_reranking = True
         return server
 
+    @staticmethod
+    def tinygemma3() -> ServerProcess:
+        server = ServerProcess()
+        # mmproj is already provided by HF registry API
+        server.model_hf_repo = "ggml-org/tinygemma3-GGUF"
+        server.model_hf_file = "tinygemma3-Q8_0.gguf"
+        server.mmproj_url = "https://huggingface.co/ggml-org/tinygemma3-GGUF/resolve/main/mmproj-tinygemma3.gguf"
```

Comment:
I trained this tiny model for using in CI, it's just 38M params

The dataset used for training was CIFAR-10 ; however, the quality is quite bad. If know how to train tiny models (less than 50M params), please reach out!